client/src/components/pages/CreatePostPage/Event/index.js

```diff
@@ -0,0 +1,248 @@
+import React from "react";
+import { InputAntd, TextAreaAntd, DropDownAntd } from "components/utils";
+import "./style.css";
```

Comment:
split the imports of the third party modules and your own and add an empty line between them